continent-countries-1.0-utf8.js

```diff
@@ -24,121 +27,131 @@ var dgContinentsCountries = function(data) {
       }
     }
   }
-  this.set(data['continent'], data['country']);
+  this.set(data.continent, data.country);
   this.start();
 
   var nome, length = keys.length;
-  for (var i=0; i<length; i++) {
+  for (var i = 0; i < length; i++) {
     nome = keys[i];
 
     if (this[nome].getAttribute('value')) {
-      data[nome+'Val'] = this[nome].getAttribute('value');
+      data[nome + 'Val'] = this[nome].getAttribute('value');
     }
 
-    if (data[nome+'Val']) {
-    var options = this[nome].options;
-    if (nome=='continent') this.continent.onchange();
-    for (var j = 0; j<options.length; j++) {
-      if (options[j].tagName == 'OPTION') {
-        if (options[j].value == data[nome+'Val']) {
-          options[j].setAttribute('selected',true);
-          if (nome=='continent'){
-            this.continent.selectedIndex=j;
-            this.continent.onchange();
+    if (data[nome + 'Val']) {
+      var options = this[nome].options;
+      if (nome == 'continent') this.continent.onchange();
+      for (var j = 0; j < options.length; j++) {
+        if (options[j].tagName == 'OPTION') {
+          if (options[j].value == data[nome + 'Val']) {
+            options[j].setAttribute('selected', true);
+            if (nome == 'continent') {
+              this.continent.selectedIndex = j;
+              this.continent.onchange();
+            }
           }
         }
       }
     }
-  }
 
   }
 
-}
+};
 
 dgContinentsCountries.prototype = {
   continent: document.createElement('select'),
   country: document.createElement('select'),
   set: function(continent, country) {
-    this.continent=continent;
-    this.continent.dgContinentsCountries=this;
-    this.country=country;
-    this.continent.onchange=function(){this.dgContinentsCountries.run()};
+    this.continent = continent;
+    this.continent.dgContinentsCountries = this;
+    this.country = country;
+    this.continent.onchange = function() {
+      this.dgContinentsCountries.run();
+    };
   },
-  start: function () {
+  start: function() {
     var continent = this.continent;
     while (continent.childNodes.length) continent.removeChild(continent.firstChild);
-    for (var i=0;i<this.continents.length;i++) this.addOption(continent, this.continents[i][0], this.continents[i][1]);
+    for (var i = 0; i < this.continents.length; i++) this.addOption(continent, this.continents[i][0], this.continents[i][1]);
+    this.addOption(country, '', 'Select a country');
   },
-  run: function () {
-  var sel = this.continent.selectedIndex;
+  run: function() {
+    var sel = this.continent.selectedIndex;
     var itens = this.countries[sel];
     var itens_total = itens.length;
 
     var opts = this.country;
     while (opts.childNodes.length) opts.removeChild(opts.firstChild);
 
-    this.addOption(opts, '', 'Select a country');
-    for (var i=0;i<itens_total;i++) this.addOption(opts, itens[i], itens[i]);
+    for (var i = 0; i < itens_total; i++) this.addOption(opts, itens[i], itens[i]);
```

Comment:
Line is too long.<br>Identifier 'itens_total' is not in camel case.<br>Expected '{' and instead saw 'this'.